src/libs/Navigation/AppNavigator/AuthScreens.tsx

```diff
@@ -271,6 +272,19 @@ function AuthScreens({session, lastOpenedPublicRoomID, initialLastUpdateIDApplie
             return;
         }
 
+        let signupQualifier;
+        if (currentUrl.includes('signupQualifier')) {
+            signupQualifier = currentUrl.split('signupQualifier=')[1].split('&')[0];
```

Comment:
Also, how about we use `new URL(currentUrl)` for an easier query params handling? You have a similar case in this file 😄 

```ts
function shouldOpenOnAdminRoom() {
    const url = getCurrentUrl();
    return url ? new URL(url).searchParams.get('openOnAdminRoom') === 'true' : false;
}
```

Reply:
This worked great!